cli/lib/kontena/cli/nodes/labels/add_command.rb

```diff
@@ -2,17 +2,17 @@ module Kontena::Cli::Nodes::Labels
   class AddCommand < Kontena::Command
     include Kontena::Cli::Common
 
-    parameter "NODE_ID", "Node id"
+    parameter "NODE", "Node"
```

Comment:
Not necessarily, because it can be either the node `name`, or the `node_id` - both of these will work.

Using the `node_id` is is only really relevant for odd cases like `kontena node rm 44C7:P5OM:NBJT:WXHV:6EDU:67T5:YDMX:4YPU:PF6D:VUH5:7LE7:5RC7`, if the agent has connected but never sent the node info... there is no name to remove it with. This reqiures picking out the node_id from `DEBUG=api kontena node list`, though :P